roles/openshift_gcp/tasks/dynamic_inventory.yml

```diff
@@ -1,5 +1,12 @@
 ---
 - name: Extract PEM from service account file
-  copy: content="{{ (lookup('file', openshift_gcp_iam_service_account_keyfile ) | from_json ).private_key }}" dest=/tmp/gce.pem mode=0600
+  copy:
+    content: "{{ (lookup('file', openshift_gcp_iam_service_account_keyfile ) | from_json ).private_key }}"
+    dest: /tmp/gce.pem
+    mode: 0600
```

Comment:
mode: 0600 was overly strict when I ran this locally (uncontainerized). Modules trying to open the file would have permission denied.

Reply:
@patrickdillon This is possibly due to incorrect use of 'become'.